src/functions-and-arrays.js

```diff
@@ -1,41 +1,113 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2){
+    return num1
+  } else if  ( num2 > num1 ){
+    return num2
+  } else {
+    return num1
+  }
+
+}
```

Comment:
Esta perfecto!!!!
Ahora una cosilla fijate que solo existe una forma de devolver num2, para todos los demas casos es num1, entonces podriamos hacer esto:
```suggestion
function maxOfTwoNumbers(num1, num2) {
  if  ( num2 > num1 ){
    return num2
  } else {
    return num1
  }
}
```